en/docs/get-started/tutorials/create-your-first-integration.md

```diff
@@ -0,0 +1,134 @@
+
+14. Click **Create**.
+
+The micro integrator REST API opens on a separate page.
+
+## Step 2: Deploy
+
+Let's deploy the micro integrator REST API you created to the developer environment by following the steps given below:
+
+1. In the left navigation menu, click **Deploy**.
+
+2. Click **Deploy Manually**.
+
+    !!! info
+        Automatic deployment is enabled for the component by default. You are required to carry out only the first deployment manually.
+
+Once Choreo has deployed the micro integrator REST API, you can proceed to the next step to test it.
+
+## Step 3: Test
+
+Choreo assists you to generate a cURL command to invoke the micro integrator REST API you created. To generate this cURL command and issue it, follow these steps:
+
+1. In the left navigation menu, click **Test**.
+
+2. In the **Method** list, select **Get**.
+
+3. Enter `/HelloWorld` as the path.
+
+4. To copy the cURL command, click the icon for copying
+
+    !!! note
+        Always use the icon to copy the cURL command instead of copying it manually because the displayed cURL command is only for display purposes and does not include the API token.
+
+5. Issue the cURL command that you copied. You will get the following response.
+
+    `{"Hello":"Integration"}`
+
+The above response indicates that your micro integrator REST API is working as expected.
+
+## Step 4: Observe
+
+To observe statistics for your micro integrator REST API, click **Observe** in the left navigation menu. The graphs for throughput and latency will display the request you sent in [Step 3: Test](#step-3-test).
+
+For more information about observing components, see [Observability Overview](../../observe-and-analyze/observe/observability-overview.md).
+
+## Step 5: Manage
+
+For a micro integrator REST API, you can perform the following API management actions:
+
+- Manage the lifecycle: You can publish your API to the Developer Portal to make it available for subscriptions by other applications, or deploy it as a prototype so that application developers can try it out and provide feedback for improvement. Once you publish an API, you can deprecate it or block it from being used if required.
+
+- Validate subscriptions: You can configure subscription validation to mandate subscriptions.
+
+- Add documents: This involves attaching files with information about the API for users.
+
+- Select/switch usage plans: You can select a usage plan for your API out of Bronze Silver, Gold, and Unlimited based on the level of traffic that you expect the API to receive. You can change the usage plan when required.
+
+- Edit properties: You can enter/update the names and email addresses of the business owner and the technical owner of the API.
+
+Let's publish the micro integrator REST API to make it available for subscriptions in the Developer Portal.
```

Comment:
Shall we add a section to this? Publish API?

Reply:
Made Step 5 the **Publish** section and provided a link to the API management page in an information note.